src/screens/WebsocketResearchScreen/SampleChatInfoScreen.tsx

```diff
@@ -153,17 +164,59 @@ export const styles = StyleSheet.create({
   textAct: {
     color: '#FF2E63',
     fontSize: 14
+  },
+  mr7: {
+    marginRight: dimen.normalizeDimen(12)
+  },
+  imageUser: {
+    height: normalize(48),
+    width: normalize(48),
+    borderRadius: normalize(24)
+  },
+  parentContact: {
+    height: normalize(72)
   }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

New styles have been added. Ensure that these are used correctly and do not affect the layout in an undesired way.



<!-- This is an auto-generated comment by OSS CodeRabbit -->